apps/examples/webclient/webclient_main.c

```diff
@@ -211,11 +263,18 @@ int webclient_init_request(void *arg, struct http_client_request_t *request)
 		request->method = WGET_MODE_DELETE;
 	} else {
 		dump_webclient_usage();
-		return -1;
+		goto exit;
 	}
 
 	/* argument2 is url. */
 	request->url = argv[2];
+	request->url = (char *)malloc(strlen(argv[2]) + 1);
```

Comment:
It looks strange. After giving some pointer at line 270. Immediatly refresh by malloc. I think you should remove line 270 because of line 275